scripts/Rakefile

```diff
@@ -73,9 +73,20 @@ task :install_pe do
     f.write ERB.new(File.read('../templates/answers.erb')).result(binding)
   end
   %x{mkdir /tmp/puppet-enterprise}
-  %x{tar xf /tmp/puppet-enterprise.tar.gz -C /tmp/puppet-enterprise --strip-components=1} 
-  puts "Installing Puppet Enterprise"
-  %x{/tmp/puppet-enterprise/puppet-enterprise-installer -D -a /tmp/answers}
+  %x{tar xf /tmp/puppet-enterprise.tar.gz -C /tmp/puppet-enterprise --strip-components=1}
+  if Gem::Version.new(PE_VERSION) < Gem::Version.new('2016.2.0')
+    File.open('/tmp/answers','w') do |f|
+      f.write ERB.new(File.read('../templates/answers.erb')).result(binding)
+    end
+    puts "Installing Puppet Enterprise with answers file for version #{PE_VERSION}"
+    %x{/tmp/puppet-enterprise/puppet-enterprise-installer -D -a /tmp/answers}
+  else
+    File.open('/tmp/pe.conf','w') do |f|
+      f.write ERB.new(File.read('../templates/pe.conf.erb')).result(binding)
```

Comment:
If need be, the answers file is an ERB, so I used the same thing.  I was trying to keep the delta small since we don't currently have any static files.